newIDE/app/scripts/import-zipped-editor.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * This script will download and extract a zipped version of a prebuilt external html5 editor
```

Comment:
Good idea to make this script generic :)

Reply:
thank you, I decided that it would be more useful than just copying the code you had in import-piskel-editor :) 
It will come in handy for bassoon too